packages/v4/patternfly-docs/patternfly-docs.source.js

```diff
@@ -38,16 +38,19 @@ module.exports = (sourceMD, sourceProps) => {
   const reactChartsPath = require
       .resolve('@patternfly/react-charts/package.json')
       .replace('package.json', 'src');
+
   const reactLogViewerPath = require
-      .resolve('@patternfly/react-log-viewer/package.json')
-      .replace('package.json', 'src');
+      .resolve('@ausuliv/react-log-viewer/package.json')
+  const logViewerContentBase = path.join(reactLogViewerPath.replace('package.json', 'patternfly-docs'), '/content/extensions/react-log-viewer')
+  const logViewerPropsBase = reactLogViewerPath.replace('package.json', 'src')
+  const logViewerPropsIgnore = ['**/*.test.tsx', '**/examples/*.tsx'];
```

Comment:
Why do we need `logViewerPropsIgnore` here? It's totally the same as `reactPropsIgnore`.

Reply:
Ah yes, looks like this probably isn't needed to me as well. I had copied this element of the approach from Cooper's quickstarts docs PR.